src/themes/types.ts

```diff
@@ -10,6 +10,82 @@ import { Extendable, ObjectOf, ObjectOrFunc } from '../../types/utils'
 //
 // We use these terms in typings to indicate which phase the typings apply to.
 
+// ========================================================
+// Colors
+// ========================================================
+
+/**
+ * A type for a palette for a single color.
+ */
+export type ColorVariants = {
+  50: string
+  100: string
+  200: string
+  300: string
+  400: string
+  500: string
+  600: string
+  700: string
+  800: string
+  900: string
+}
+
+/**
+ * A type for a predefined natural colors.
+ */
+type NaturalColorsStrict = Partial<{
+  blue: ColorVariants
```

Comment:
frankly, still have question whether we should predefine this set - essentially, this would imply that we will _request_ this set to be defined by the client, even in case if only two colors are intended to be used by the app. We should think about making this process as smooth as possible for the client, as color palette may be far from being the main concern of the one who has decided to use Stardust (and, potentially, other important features of theming)

Reply:
This type is not exported, we export `NaturalColors` which is extendable. This allows you to get autocomplete on frequent colors and define yours.